pbft/test/database_pbft_service_test.cpp

```diff
@@ -91,7 +92,11 @@ TEST(database_pbft_service, test_that_executed_operation_fires_callback_with_ope
 
     EXPECT_CALL(*mock_io_context, post(_)).WillOnce(InvokeArgument<0>());
 
-    bzn::database_pbft_service dps(mock_io_context, mem_storage, mock_crud, TEST_UUID);
+    auto mock_monitor = std::make_shared<bzn::mock_monitor>();
+
+    EXPECT_CALL(*mock_monitor, finish_timer(bzn::statistic::request_latency, "somehash"));
```

Comment:
Cool, somewhere there is a map that must have the time that the request was sent...